frontend/app/src/composables/locations.ts

```diff
@@ -56,37 +23,35 @@ export const useLocations = createSharedComposable(() => {
   const { getAssetImageUrl } = useAssetIcon();
   const { getChainInfoById } = useSupportedChains();
 
-  const getLocationData = (identifier: MaybeRef<string>): TradeLocationData => {
-    const id = get(identifier);
-    if (isBlockchain(id)) {
-      const assetId = getIdentifierFromSymbolMap(id);
-
-      return {
-        name: get(getChainInfoById(id))?.name || id,
-        identifier: assetId,
-        image: getAssetImageUrl(assetId),
-        detailPath: `${Routes.ACCOUNTS_BALANCES_BLOCKCHAIN}#blockchain-balances-${id}`
-      };
-    }
-
-    const locationFound = get(tradeLocations).find(
-      location => location.identifier === id
-    );
+  const locationData = (
```

Comment:
Btw I wonder why the base function here is the one that returns computed (`locationData`)
and the `getLocationData` need to call that function
Why not the other way around? Since we can reduce the computed call. Is it some ref issue?

Reply:
Okay I think I got it, if I do the other way, the `getChainInfoById` won't be reactive, since it's not inside computed directly